src/deck-options.md

```diff
@@ -2,262 +2,223 @@
 
 <!-- toc -->
 
-Deck options primarily control the way Anki schedules cards. It is recommended
+Deck options primarily control the way Anki schedules cards. If you're a new user, it's recommended
 that you spend a few weeks with the defaults to get a feel for how Anki works
-before you start adjusting options. Please make sure you understand the options
+before you start adjusting the options. Please make sure you understand the options
 before changing them, as mistakes could reduce Anki's effectiveness.
 
-Deck options are accessed by:
+If you're on the desktop app, do any of the following to access deck options:
 
-- Clicking the gear icon on the `Decks` screen.
-- Selecting a deck on the `Decks` screen, and then clicking `Options`
+- Click the gear icon on the Decks screen.
+- Select a deck on the Decks screen, and then click "Options"
   at the bottom of the screen.
-- Clicking on `More` > `Options` while in review mode.
-- Pressing `o` while in review mode.
+- Click "More" > "Options" while in review mode.
+- Press <kbd>o</kbd> while in review mode.
 
-This page describes the options shown in Anki 2.1.45+, when you have the v2 or
-v3 scheduler enabled. On older versions, some options will not be available, or
-will appear in a different section. Please keep in mind that the v1 scheduler
-is no longer supported in Anki 2.1.50+. If you have not yet updated to v2 or v3,
-you will be prompted to update when you attempt to review cards in 2.1.50+.
-
-For more info on deck options, please check:
+Here are some community posts about deck options:
 
 - [Deck Options Explained](https://forums.ankiweb.net/t/deck-options-explained/213)
 - [Deck Options in a Mental Map](https://forums.ankiweb.net/t/deck-options-in-a-mental-map/15757)
 
+Note: both of them were written for the earlier versions of Anki.
```

Comment:
I don't think that's accurate or needed here. While "Explained" was published in June 2020 -- circa 2.1.26 -- the Mental Map was published in Dec 2021 -- circa 2.1.49 -- and updated substantively in 2023. It definitely doesn't predate 2.1.45.

If there are problems with either one of them, let's talk about how to address that in brief, but don't just equate old with bad.

Reply:
If you've noticed, I always write "earlier" and never "old". Earlier isn't bad. It's just useful to know that they are written for earlier versions, as that is something people never presume on their own.

I'm not sure whether you're saying the first post is still mostly updated, I think the sections were for earlier Anki versions. As for cqg's post, I remember I found some missing info there, and it was updated last in 2023.

Reply:
What I'm saying is (1) compared to the fixed reference point of 2.1.45, these aren't necessarily "earlier," and (2) you should know what's in them before you change how they are described here.

Reply:
1. I was never made aware of this terminology. I have been using earlier to mean anything not from this time.
2. I've read those.

Reply:
Also not resolved.